src/levels/remote/lockedMaster.js

```diff
@@ -19,7 +19,7 @@ exports.level = {
     "en_US": "Locked Master",
     "de_DE": "Gesperrter Master-Branch",
     "ru_RU": "Заблокированная ветвь master",
-    "ru_RU": "Заблокована гілка master",
+    "uk": "Заблокована гілка master",
```

Comment:
hrm maybe the git diff is just rendering weird but shouldnt this be removing the `uk` entry? its showing up as if its still mapped to the russian translation